SPECS/mstflint/mstflint.spec

```diff
@@ -40,13 +56,33 @@ find . -type f -iname '*.cpp' -exec chmod a-x '{}' ';'
 %configure --enable-fw-mgr --enable-adb-generic-tools
 %make_build
 
+# Compile mstflint driver
+export EXTRA_CFLAGS='-DVERSION=\"%version\"'
+cd kernel
+set -- *
+mkdir source
+mv "$@" source/
+mkdir obj
+
+rm -rf obj
+cp -a source obj
+cd $PWD/obj
+make KPVER=%{kver}
```

Comment:
I'm somewhat confused by this build procedure. It seems we're first copying the contents of `kernel` into the `source` subdirectory and then again copy that to `obj` (which may have been deleted a step earlier) just to call `make KPVER=%{kver}` from inside `obj`. I think I am missing something here. Couldn't we just call `EXTRA_CFLAGS='-DVERSION=\"%version\"' make -C kernel KPVER=%{kver}` and skip all the extra steps?

```suggestion
# Compile mstflint driver
EXTRA_CFLAGS='-DVERSION=\"%version\"' make -C kernel KPVER=%{kver}
```

Reply:
Thanks for the suggestion. Applied all except the -C option, as mstflint kernel Makefile is dependent on {pwd} and it doesn't work if it's not issued inside the directory.